lib/documents.js

```diff
@@ -230,7 +225,8 @@ export async function _getRegistrationRecord(
   };
   const projection = {_id: 0};
   const collection = database.collections['tokenization-registration'];
-  const record = await collection.findOne(query, {projection});
+  const record = await collection.findOne(
+    query, {projection, promoteBuffers: true});
```

Comment:
with bedrock-mongodb @8.0.1 `promoteBuffers` is not required, is that right?

Reply:
Yeah, I just missed this one. Fixed, thanks.